openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/hiveConnection.json

```diff
@@ -15,7 +15,7 @@
     "hiveScheme": {
       "description": "SQLAlchemy driver scheme options.",
       "type": "string",
-      "enum": ["hive", "hive+http", "hive+https", "impala", "impala4"],
+      "enum": ["hive", "hive+http", "hive+https"],
```

Comment:
@pmbrull we should add migration for this right? for those who might have created the service with impala

Reply:
you are so right, thank you